pkg/build/builder/cmd/builder.go

```diff
@@ -372,7 +370,6 @@ func RunGitClone(out io.Writer) error {
 // and also adds some env and label values to the dockerfile based on
 // the build information.
 func RunManageDockerfile(out io.Writer) error {
-	serviceability.InitLogrus("DEBUG")
```

Comment:
@bparees ditto here - was this intended to be kept when you fixed the debug logging for buildah?

Reply:
hm.  I don't think so, at least i can't think of why we'd want it like this.